bam.iobio/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+	"compilerOptions": {
+		"target": "es5",
+		"lib": ["dom", "dom.iterable", "esnext"],
+		"allowJs": true,
+		"skipLibCheck": true,
+		"alwaysStrict": true /* Ensure 'use strict' is always emitted. */,
+		"strict": true /* Enable all strict type-checking options. */,
+		"forceConsistentCasingInFileNames": true,
+		"noEmit": true,
+		"esModuleInterop": true,
+		"module": "esnext",
+		"moduleResolution": "node",
+		"resolveJsonModule": true,
+		"isolatedModules": true,
+		"jsx": "preserve",
+		"incremental": true,
+		"noImplicitAny": true /* Enable error reporting for expressions and declarations with an implied `any` type.. */,
+		"strictNullChecks": true /* When type checking, take into account `null` and `undefined`. */,
+		"baseUrl": "./",
+		"paths": {
+			"@/public": ["./public"],
+			"@/*": ["./*"],
+			"react": ["./node_modules/@types/react"]
```

Comment:
I see this line in other overture-stack projects. I'm confused how this `react` alias doesn't mess up imports in modules. eg. `import useEffect from 'react'` - any idea?

Reply:
I'm investigating, but my best guess is this is legacy code from before TS was widely adopted

You can read in TS Docs that `paths` does not affect actual emitted code, which I think is why it doesn't mess up imports: https://www.typescriptlang.org/docs/handbook/modules/reference.html#paths

And you can find old issues related to this line suggesting it may be a workaround that is no longer needed: https://github.com/microsoft/TypeScript/issues/9566

Reply:
sweet thank you!

Reply:
that's correct. this line is not needed anymore
```suggestion
```